lib/SPIRV/SPIRVReader.cpp

```diff
@@ -300,7 +314,8 @@ bool SPIRVToLLVM::transOCLBuiltinFromVariable(GlobalVariable *GV,
   } else {
     FuncName = std::string(GV->getName());
   }
-  Type *ReturnTy = GV->getType()->getPointerElementType();
+  Type *GVTy = GV->getType()->getPointerElementType();
+  Type *ReturnTy = GVTy;
```

Comment:
This split is done just to improve code readability, correct? I don't see any GVTy/ReturnTy modifications below.

Reply:
No, not only for this. ReturnTy is re-initialized later in the code. I needed to keep original type of the global variable.

Reply:
Ah, it's line 325, I've missed it.